runtime/src/attestation/mod.rs

```diff
@@ -84,7 +84,7 @@ decl_module! {
 						return Self::error(Self::ERROR_DELEGATION_NOT_AUTHORIZED_TO_ATTEST);
 					} else {
 						// check if CTYPE of the delegation is matching the CTYPE of the attestation
-						let root = <delegation::Root<T>>::get(delegation.0.clone());
+						let root = <delegation::Root<T>>::get(delegation.0.clone()).ok_or(<delegation::Module<T>>::ERROR_ROOT_NOT_FOUND.1)?;
```

Comment:
We need to change this. I will talk to you in person, but we need to use the error module...